### DISCUSSION TITLE
az181996w@gmail.com

### DISCUSSION THREAD

**Discussion by Mahmoudessam807** [community/community#147943] [GitHub Education]
### Hi everyone,

مشكلة الدفع والسحب

**Comment by Ravindulakmina** [comment]
hi

**Comment by ebndev** [comment]
Hi @Mahmoudessam807, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
